src/cloudai/configurator/base_gym.py

```
@@ -67,12 +67,13 @@ def reset(
         pass
 
     @abstractmethod
-    def step(self, action: Any) -> Tuple[list, float, bool, dict]:
+    def step(self, action: Any, constraint_check_reward: float) -> Tuple[list, float, bool, dict]:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `action`

(ANN401)